api/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path, re_path
+from rest_framework import routers
 
-from . import views
+from api.views.song import SongViewSet
 
-urlpatterns = [
-    re_path(r"", views.test_view),
-]
+router = routers.DefaultRouter()
+
+router.register(r"songs", SongViewSet)
```

Comment:
This links the code for the endpoint to the actual URL (note that all URLs in this file are relative to `/api/...`). We'll also need to connect the (yet to be made) SongbookViewSet and SongEntryViewSet here as well. (e.g. we'll need one more line for each of those ViewSets).